src/others/xml/lv_xml_component.c

```diff
@@ -54,6 +57,24 @@ static lv_ll_t component_ctx_ll;
 void lv_xml_component_init(void)
 {
     lv_ll_init(&component_ctx_ll, sizeof(lv_xml_component_ctx_t));
+
+    lv_xml_component_ctx_t * global_ctx = lv_ll_ins_head(&component_ctx_ll);
+    lv_memzero(global_ctx, sizeof(lv_xml_component_ctx_t));
+    lv_xml_component_ctx_init(global_ctx);
+    global_ctx->name = lv_strdup("globals");
+
+}
+
+void lv_xml_component_ctx_init(lv_xml_component_ctx_t * ctx)
+{
+    lv_ll_init(&ctx->style_ll, sizeof(lv_xml_style_t));
```

Comment:
these init functions may fail? If so how they are being handled? Is there any indication for user when something goes wrong?

Reply:
That is okay, even though they failure chance is low, if that occurs, it will be silent, but having proper logs or error code the failure here, if detected can point the user to investigate the root cause.

It is not a blocker at all, feel free to resolve :)

Reply:
Resolving for now then. 🙂